.github/workflows/test.yml

```diff
@@ -28,32 +28,49 @@ jobs:
     strategy:
       matrix:
         go-version: [1.14.x]
-        platform: [ubuntu-latest]
-    runs-on: ${{ matrix.platform }}
+        os: [ubuntu-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Install Go
         uses: actions/setup-go@v1
         with:
           go-version: ${{ matrix.go-version }}
+      - name: Enable long path support for git # sigh
+        if: ${{ runner.os == 'Windows' }}
+        run: git config --global core.longpaths true
```

Comment:
Oh wow, I didn't realise that was a git-on-windows error!

Reply:
yep!